resources/assets/js/components/importer/importer-file.vue

```diff
@@ -266,7 +266,7 @@
                 }
                 this.statusType='pending';
                 this.statusText = "Processing...";
-                this.$http.post(route('api.imports.importFile', this.file.id), {
+                this.$http.post('api/v1/imports/process/' + this.file.id, {
```

Comment:
Yeah, it’s weird because I’m in raw JS - but I think we have some weird meta `baseUrl` something or other I can consume for this (and the others). Fair point.